simulation/src/test/resources/report.md

```diff
@@ -25,23 +25,23 @@
                 one_big_spike[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT2.667606696S 	server_cpu=PT2M30S        	client_received=1000/1000	server_resps=1000	codes={200=1000}
                     one_big_spike[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT1.579726259S 	server_cpu=PT2M30S        	client_received=1000/1000	server_resps=1000	codes={200=1000}
                               one_big_spike[UNLIMITED_ROUND_ROBIN].txt:	success=99.9%	client_mean=PT1.000523583S 	server_cpu=PT7M37.35S     	client_received=1000/1000	server_resps=3049	codes={200=999, 429=1}
-one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=63.9%	client_mean=PT9.2169728S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1597, 500=903}
- one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=64.1%	client_mean=PT4.6030464S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1602, 500=898}
+one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=66.0%	client_mean=PT2.0950112S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1649, 500=851}
+ one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=66.2%	client_mean=PT2.0278288S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1655, 500=845}
```

Comment:
Oh this is pretty 😍 - requests keep flowing to the working endpoints and are properly throttled to the broken ones!

![image](https://user-images.githubusercontent.com/3473798/94922714-038e8780-04bb-11eb-9d92-1b942d5df9a8.png)

Reply:
It works better -- it's not the ideal design for this scenario because we don't take the endpoint into account when we make a node selection decision, but I think it will save us some headache :-)